apollo/resolvers/user/createUser.ts

```diff
@@ -0,0 +1,15 @@
+import bcrypt from 'bcryptjs';
+
+interface CreateUserArgs {
+  username: string;
+  password: string;
+}
+
+export const createUser = async (
+  _: Record<string, never>,
+  { username, password }: CreateUserArgs,
+) => {
+  const hash = await bcrypt.hash(password, 10);
+
+  return `${username}:${hash}`;
+};
```

Comment:
If we're feeling lazy, we can strip out all of the create user, login, register etc. from another project of mine, but it's all pretty straight forward

Reply:
login is covered by next-auth - we just need create user afaik (and maybe update profile and stuff eventually)